docs/developer/community/contributing.mdx

```diff
@@ -37,6 +34,163 @@ For compatibility, Saleor also provides `requirements.txt` and `requirements_dev
 
 These files should be updated by running `poetry export --without-hashes -f requirements.txt -o requirements.txt` and `poetry export --without-hashes -f requirements.txt -o requirements_dev.txt --dev`, respectively.
 
+## File structure
+We are using a standard Django structure - every app has its own directory, where you can find:
+- migrations directory
+- management directory
+- models
+- utils - that keeps some functions of general utility related to this module
+- error codes - the definitions of errors that might appeared
+- tests directory - that contains related tests,
+- etc.
+
+### API file structure
+The `API` files are in `saleor/graphql/` directory. Every app has its own directory 
+inside with:
+- `schema.py` - with definitions of queries and mutations
+- `sorters.py` - with definitions of sorters
+- `filters.py` - with definitions of filters
+- `types.py` - with definitions of corresponding types
+- `enums.py` - with related enums
+- `dataloaders.py` - with data loaders for the given module.
+- mutations file or directory.
+- test directory.
+
+We are aiming to have `mutations` directory in every module with a file per every mutation
+how it's done in the `checkout` directory. See an example below:
+
+```bash
+.
+└── saleor
+    └── graphql
+        └── checkout
+            ├── migrations
+            │   └── checkout_create.py
+            │   └── checkout_complete.py
+            └── schema.py
+```
+
+### Tests file structure
+We are keeping tests in `tests` directory. We are aiming to keep queries and mutations
+in separate directories with a single file for every query and mutation. 
+After joining it with the previous example, it would look like this:
+
+```bash
+.
+└── saleor
+    └── graphql
+        └── checkout
+            ├── migrations
+            │   └── checkout_create.py
+            │   └── checkout_complete.py
+            ├── schema.py
+            └── tests
+                ├── migrations
+                │   ├── test_checkout_complete.py
+                │   └── test_checkout_create.py
+                └── queries
+                    └── test_checkout.py
```

Comment:
Maybe we could add here:
```
queries:
 - test_checkout.py
 - test_checkouts.py
 ```
 Just to emphasize, we want to have  separate files for `query checkout(id:<id>)` and for `query checkouts(first:10)`